src/results.js

```diff
@@ -45,7 +45,7 @@ class Results extends React.Component {
       var indexCounts = this.props.collections.map((col) => {
           if (this.props.counts.hasOwnProperty(col)) {
             return (
-              <span>
+              <span key={Math.random().toString(36).substring(3)}>
```

Comment:
Can we generate a key based on the values in `col`? Using random strings basically ensures React will have to re-render every subitem every time.